index.js

```diff
@@ -156,27 +157,64 @@ module.exports = {
     "space-unary-ops": "error",
 
     // enforce consistent spacing after the // or /* in a comment, and before the */
-    "spaced-comment": ["error", "always", { "line": { "markers": ["/"]}, "block": { "balanced": true}}],
+    "spaced-comment": ["error", "always", { "line": { "markers": ["/"] }, "block": { "balanced": true } }],
 
     // require the jsdocs to be valid
     "valid-jsdoc": ["error", {
       "requireParamDescription": false,
       "requireReturnDescription": false,
 
       // only require a return if there is one in the function
-      "requireReturn": false,
+      "requireReturn": false
+    }],
 
+    // Rules from jsdoc plugin
+    "jsdoc/check-alignment": 1, // Recommended
+    "jsdoc/check-examples": 0,
+    "jsdoc/check-indentation": 0,
+    "jsdoc/check-param-names": 1, // Recommended
+    "jsdoc/check-syntax": 0,
+    "jsdoc/check-tag-names": 1, // Recommended
+    "jsdoc/check-types": 1, // Recommended
+    "jsdoc/implements-on-classes": 1, // Recommended
+    "jsdoc/match-description": 0,
+    "jsdoc/newline-after-description": 1, // Recommended
+    "jsdoc/no-types": 0,
+    // if this is enabled, all global namespaces (goog, ol, os, etc) must be defined in the config
+    "jsdoc/no-undefined-types": 0, // Recommended
+    "jsdoc/require-description": 0,
+    "jsdoc/require-description-complete-sentence": 0,
+    "jsdoc/require-example": 0,
+    "jsdoc/require-hyphen-before-param-description": 0,
+    "jsdoc/require-jsdoc": 1, // Recommended
+    "jsdoc/require-param": 1, // Recommended
+    "jsdoc/require-param-description": 0, // Recommended
+    "jsdoc/require-param-name": 1, // Recommended
+    "jsdoc/require-param-type": 1, // Recommended
+    // only require a return if there is one in the function
+    "jsdoc/require-returns": 0, // Recommended
```

Comment:
I think we want this one enabled. According to [the documentation](https://www.npmjs.com/package/eslint-plugin-jsdoc#require-returns), this rule verifies functions with a return are documented. It doesn't require all functions to have a return.

Reply:
Fixed. I turned off each of the recommended rules only when they were problematic, but this may have been a false alarm before getting another setting right. In any case, checked it is OK now.